var/spack/repos/builtin/packages/py-lazyarray/package.py

```diff
@@ -13,8 +13,12 @@ class PyLazyarray(PythonPackage):
     homepage = "http://bitbucket.org/apdavison/lazyarray/"
     pypi = "lazyarray/lazyarray-0.2.8.tar.gz"
 
+    version('0.3.2',  sha256='be980534c5950a976709085570f69be9534bdf0f3e5c21a9113de3ee2052683e')
     version('0.2.10', sha256='7a53f81b5f3a098c04003d2ad179fc197451fd96bc921510f8534c6af8cc8e19')
     version('0.2.8',  sha256='aaee4e18117cc512de7a4e64522f37bc6f4bf125ecffdbdbf4e4e390fbdd9ba2')
 
+    depends_on('python@2.7:', when='@0.3:')
     depends_on('py-numpy@1.3:', type=('build', 'run'))
+    depends_on('py-numpy@1.8:', type=('build', 'run'), when='@0.3:')
     depends_on('py-numpy@1.5:', type=('build', 'run'), when='^python@3:')
+    depends_on('py-numpy@1.12:', type=('build', 'run'), when='@0.3:^python@3:')
```

Comment:
Where do you see this requirement?

Reply:
see above (and yes they are not the best at documenting stuff in machine readable form)